tools/bigquery-zos-mainframe-connector/USING_WITH_BQ.md

```diff
@@ -0,0 +1,227 @@
+# Using with BigQuery
+
```

Comment:
add notes about the typical workflow being uploading a mainframe dataset, transcoding to ORC, then registering as an external table, then running a MERGE DML statement to load a daily incremental dataset into a target table

Reply:
added